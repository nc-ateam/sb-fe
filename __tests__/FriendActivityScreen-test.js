import React from "react";
import "react-native";
import FriendActivityScreen from "../screens/FriendActivityScreen";

import renderer from "react-test-renderer";

describe("<FriendActivityScreen />", () => {
  it("renders without crashing", () => {
    const rendered = renderer.create(<FriendActivityScreen />).toJSON();
    expect(rendered).toBeTruthy();
  });

  it("creates a screenshot", () => {
    const rendered = renderer.create(<FriendActivityScreen />).toJSON();
    expect(rendered).toMatchSnapshot();
  });
});
