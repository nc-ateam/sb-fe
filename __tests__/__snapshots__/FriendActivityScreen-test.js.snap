// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FriendActivityScreen /> creates a screenshot 1`] = `
<View
  animationOptions={Object {}}
  onLayout={[Function]}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "white",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    animationOptions={Object {}}
    ellipsizeMode="tail"
    onLayout={[Function]}
    style={
      Object {
        "backgroundColor": "transparent",
        "color": "#666666",
        "fontFamily": "Rubik-Regular",
        "fontSize": 15,
        "fontStyle": "normal",
        "fontWeight": "normal",
      }
    }
  >
    Coming soon!
  </Text>
  <View
    style={
      Object {
        "backgroundColor": "#FFFFFF",
        "borderBottomColor": "#BDBDBD",
        "borderBottomWidth": 1,
        "bottom": 0,
        "height": 70,
        "left": 0,
        "paddingTop": 20,
        "position": "absolute",
        "right": 0,
        "top": 0,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "flex-end",
          "backgroundColor": "transparent",
          "flex": 1,
          "flexDirection": "row",
          "justifyContent": "space-between",
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "flex-start",
            "alignSelf": "center",
            "flex": 1,
          }
        }
      >
        <Text
          accessible={true}
          allowFontScaling={false}
          animationOptions={Object {}}
          ellipsizeMode="tail"
          onLayout={[Function]}
          onPress={[Function]}
          style={
            Array [
              Object {
                "color": undefined,
                "fontSize": 12,
              },
              Object {
                "backgroundColor": "transparent",
                "color": "#222222",
                "fontSize": 24,
                "paddingLeft": 9,
                "paddingRight": 9,
                "textAlign": "center",
                "textAlignVertical": "center",
              },
              Object {
                "fontFamily": "rubicon-icon-font",
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
            ]
          }
        >
          î¤‰
        </Text>
      </View>
      <View
        style={
          Object {
            "alignItems": "center",
            "alignSelf": "center",
            "flex": 1,
            "marginBottom": 0,
          }
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          animationOptions={Object {}}
          ellipsizeMode="tail"
          onLayout={[Function]}
          style={
            Object {
              "backgroundColor": "transparent",
              "color": "#222222",
              "fontFamily": "Rubik-Regular",
              "fontSize": 15,
              "fontStyle": "normal",
              "fontWeight": "normal",
              "lineHeight": 18,
            }
          }
        >
          Friend Activity
        </Text>
      </View>
      <View
        style={
          Object {
            "alignItems": "flex-end",
            "alignSelf": "center",
            "flex": 1,
          }
        }
      />
    </View>
  </View>
</View>
`;
